src/Pos/Context/Functions.hs

```diff
@@ -4,6 +4,7 @@ module Pos.Context.Functions
        (
          -- * Genesis
          GenesisUtxo(..)
+       , mkGenesisTxpContext
```

Comment:
I think `Context` re-exports `GenesisUtxo` because it was initially defined there and then someone didn't want to change imports in many places.
Anyway, that's not the main reason I rejected this PR, if it was the only one I would approve because it's not so important.